docs/howtoguides.rst

```diff
@@ -104,10 +104,15 @@ Ubuntu Pro Client for Clouds
    Create a customised Cloud Ubuntu Pro image <howtoguides/create_pro_golden_image>
    Cloud Ubuntu Pro images with FIPS updates <howtoguides/create_a_fips_updates_pro_cloud_image>
 
-Fixing multiple CVEs
-====================
+CVE/USN resolution
```

Comment:
With headers (especially in how-tos) I tend to try and use language that feels a bit more "active" because it reflects the sort of keywords users might use when they're searching the docs for something. In this case I'd recommend switching this to something more like "Resolve CVEs and USNs" or "Fixing CVEs and USNs". I'll let you have a think and decide what wording you want to use, I would just always avoid using words ending in "-tion" because they don't come up in searches very often.

Reply:
Now that I think of it, makes much more sense to have something like "Fixing multiple CVEs and USNs"